projects/cdk/tokens/active-element.ts

```diff
@@ -35,8 +35,11 @@ export const TUI_ACTIVE_ELEMENT = new InjectionToken<Observable<EventTarget | nu
 
             return merge(
                 focusout$.pipe(
+                    // eslint-disable-next-line rxjs/no-unsafe-takeuntil
                     takeUntil(mousedown$),
-                    repeatWhen(() => mouseup$),
+                    repeatWhen(notifications =>
```

Comment:
Can you explain the difference?

Reply:
This is recommended rule 

<img width="1078" alt="image" src="https://user-images.githubusercontent.com/12021443/162406905-a4c1ad4b-79f3-4759-a4ac-c87cd97c419e.png">

it also revealed an incorrect e2e test

Reply:
I just don't know how this notifier works :)